LICENCE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
+
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you add the `Year` and your `full name` at the showed `placeholder` above.